src/azure-cli/azure/cli/command_modules/keyvault/_validators.py

```diff
@@ -166,6 +166,27 @@ def validate_policy_permissions(ns):
             '--certificate-permissions --storage-permissions')
 
 
+def validate_private_endpoint_connection_id(cmd, ns):
+    connection_id = ns.connection_id
+    connection_name = ns.connection_name
+    vault_name = ns.vault_name
+
+    if not connection_id:
+        if not all([connection_name, vault_name]):
+            raise argparse.ArgumentError(
+                None, 'specify both: --connection-name and --vault-name')
```

Comment:
In `az keyvault private-endpoint show -h`, it should be `--name` instead of `--vault-name`.
```suggestion
                None, 'specify both: --connection-name and --name')
```

Reply:
@jiasli I prefer to replace all `--name` to `--vault-name` as this is consistent with other subgroup like `az keyvault key`, for the subgroup, `--name/-n` should represent the name of the sub resource itself (`key`, `secret`, `private-endpoint`), the name of vault should be represented by `--vault-name`.
How do you think?

Reply:
Makes sense. In that case, it should be --vault-name here: https://github.com/Azure/azure-cli/blob/cc53d1f95e3956444b885db0f3f48c2076cd3388/src/azure-cli/azure/cli/command_modules/keyvault/_params.py#L133

Reply:
@jiasli Yea, I will change it.